xml/System.Text.Json.Serialization/JsonStringEnumConverter.xml

```diff
@@ -51,9 +59,9 @@
         <Parameter Name="allowIntegerValues" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="namingPolicy">To be added.</param>
-        <param name="allowIntegerValues">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="namingPolicy">Optional naming policy for writing enum values.</param>
+        <param name="allowIntegerValues"><see langword="true" /> to allow undefined enum values. When <see langword="true" />, if an enum value isn't defined it will output as a number rather than a string.</param>
```

Comment:
```suggestion
        <param name="allowIntegerValues"><see langword="true" /> to allow undefined enum values; otherwise, <see langword="false" />. When <see langword="true" />, if an enum value isn't defined, it will output as a number rather than a string.</param>
```